docker/.env

```
@@ -400,11 +417,14 @@ SMQ_MQTT_ADAPTER_CACHE_MAX_COST=1048576
 SMQ_MQTT_ADAPTER_CACHE_BUFFER_ITEMS=64
 
 ### CoAP
+## If enabled run make all inside docker/ssl directory to generate the DTLS certs
+SMQ_COAP_DTLS=
 SMQ_COAP_ADAPTER_LOG_LEVEL=debug
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SMQ_COAP_ADAPTER_LOG_LEVEL key should go before the SMQ_COAP_DTLS key

(UnorderedKey)